public_html/formCreator/index.php

```diff
@@ -141,6 +143,8 @@
 			$engine->openDB->escape($form['formContainer']),      // container=
 			$engine->openDB->escape($form['formProduction']),     // production=
 			$engine->openDB->escape($form['formMetadata']),       // metadata=
+			$engine->openDB->escape($form['exportPublic']),       // export public
+			$engine->openDB->escape($form['exportOAI']),          // export OAI
```

Comment:
Having the comments for consistency is fine here. but they aren't needed. on a future update, when this mess of a file is cleaned up, they should probably be removed.